src/main/java/se/sundsvall/notes/api/model/DifferenceResponse.java

```diff
@@ -0,0 +1,61 @@
+package se.sundsvall.notes.api.model;
+
+import io.swagger.v3.oas.annotations.media.ArraySchema;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+import java.util.List;
+import java.util.Objects;
+
+import static io.swagger.v3.oas.annotations.media.Schema.AccessMode.READ_ONLY;
+
+@Schema(description = "DifferenceResponse model")
+public class DifferenceResponse {
+
+
+	@ArraySchema(schema = @Schema(implementation = Event.class, accessMode = READ_ONLY))
+	private List<Event> events;
```

Comment:
The RFC is referring to operations (instead of events). 
Should we use the RFC-terminology?